contribute-to-timescaledb/contribute-to-docs.md

```diff
@@ -0,0 +1,266 @@
+are highly welcome.
+
+The contribution and style guides introduce you to the Timescale documentation
+workflow, style, and standards. But don't worry too much about perfection. The
+documentation team can help you polish your content when your pull request is
+ready. 
+
+To learn more:
+*   Keep reading this style guide, which covers the documentation's use of the
+    English language
+*   See the [contribution guide][contributors] to learn about the GitHub
+    workflows, repository structure, and markup conventions
+
+## Style guide sections
+
+This style guide covers:
+*   [Style references](#style-references)
+    *   Style references in order of importance
+*   [Language](#language)
+    *   Plain English
+    *   Tense and perspective
+    *   Voice
+    *   Grammar
+    *   User interface actions
+    *   Prose linting with Vale
+*   [Structure](#structure)
+    *   Information when it's needed, where it's needed
+    *   Documenting processes and procedures
+*   [Word use](#word-use)
+    *   Word list from A to Z
+
+## Style references
+To make style decisions, consult resources in this order:
+1.  This guide. Check this guide first, because it contains project-specific
+    guidelines that override guidance given elsewhere.
+1.  The [Google Developer Documentation Style Guide][google-style]. Contains
+    most of the relevant styling information.
+1.  The [Chicago Manual of Style][chicago-manual]. Covers formatting decisions
+    that aren't covered in other resources.
+1.  [Merriam-Webster][merriam-webster]. Timescale documentation is written in US
+    English.
+
+## Language
+
+### Plain English
+Write in plain English.
+
+Choose the simplest and clearest language to express an idea. Readers often come
+to documentation when they're stressed. They skim text, jump between paragraphs,
+skip words, and give up quickly if things seem complex. You can reduce their
+reading workload by using:
+*   Short sentences
+*   Plain language
+*   Common vocabulary
+*   A minimum number of eye-catching details such as admonitions
+
+For example, here's the same idea expressed three different ways, with varying
+clarity:
+
+|Style|Example|
+|-|-|
+|❌ Natural English|In order to perform X installation process, please ensure that all of the following steps are done ...|
+|❌ Tech writer's English|To perform the X installation process, verify you have done the following steps ...|
+|✅ Plain English|To install X, do these steps ...|
+
+### Tense and perspective
+Use simple present tense and the second person singular ("you"). Avoid the first
+person plural ("we," "us"), because it isn't clear who is included in that
+group.
+
+### Voice
+We prefer active voice, but don't be afraid to use passive when it serves
+better. For example, the most important words come first in an English sentence.
+If the object of an action is the most important part of a sentence, put it
+first, even if it means using passive.
+
```

Comment:
I have a way of explaining this where I point out that, in technical documentation, we almost always know who the actor is (it's the reader, "you"), so in most cases the action is actually the most important part of the sentence ("Install the thing", "Configure the thing", etc). Feel free to construct that into a more useful piece of advice here, if you think it's useful 😉